### PR TITLE
Adding a few other setup steps

### PR DESCRIPTION
**Pull request by marchello2000**
* mysql setup
* other minor additions

### CODE REVIEW

**Comment by marchello2000** [comment]
@robzienert fyi

File: files/mysql-setup.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+SET tx_isolation = 'READ-COMMITTED';
```
Comment (robzienert) [inline files/mysql-setup.sql]: I think this file may be better contributed to the `orca-sql-mysql` module.

**Review by robzienert — CHANGES_REQUESTED** [review]
Lovely thanks, except one thing.

File: files/mysql-setup.sql (hunk 1 above)
Comment (marchello2000) [reply files/mysql-setup.sql]: Sorry, I don't know what this means. Where is this module?

Comment (robzienert) [reply files/mysql-setup.sql]: Oops, right... https://github.com/spinnaker/orca/tree/master/orca-sql-mysql

Comment (marchello2000) [reply files/mysql-setup.sql]: fixed up, https://github.com/spinnaker/orca/pull/2627

**Comment by marchello2000** [comment]
@robzienert you cool with the updates?

**Pull request merged**